### DISCUSSION TITLE
Best way to create deeply nested data?

### DISCUSSION THREAD

**Discussion by wireva** [prisma/prisma#14428] [Q&A] [answered]
Hi,
what is the best way to create deeply nested data? I could iterate over all the data to the lowest level and wrap everything in {create: data}, but maybe there is a better way? I would like to give it some sort of option to just create the whole structure without having to add create to every level.

**Comment by pantharshit00** [comment, marked as answer]
Hello @niwla23, our apologies for the delay in getting back to you! Due to the large number of questions in our community channels, unfortunately this slipped through the cracks. It's very important for us to address your question, so let us know in a comment if this is still an open discussion. 

You can also do a createMany on each level here which can improve performance. Otherwise you do need to iterate over your data here.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
